src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -225,7 +236,7 @@ public void createCaseDataWithIncorrectCourt() {
 
         caseDetails.setCourtCode(NOT_FOUND_COURT_CODE);
 
-        ErrorResponse result = given()
+        uk.gov.justice.probation.courtcaseservice.controller.ErrorResponse result = (uk.gov.justice.probation.courtcaseservice.controller.ErrorResponse) given()
```

Comment:
This should be reverted to `ErrorResponse result = given()`

Reply:
Good spot...I'll sort it now